tests/constructor/configs/config.yaml

```diff
@@ -0,0 +1,76 @@
+# defaults:
```

Comment:
What does this comment mean?

Reply:
Same reason as ```'base_' + config_name```

https://hydra.cc/docs/upgrades/1.0_to_1.1/automatic_schema_matching/#:~:text=base_%20prefix%2C%20e.g.%20base_mysql.